media-video/guvcview/guvcview-2.2.1.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Simple Qt6 or GTK+3 interface for capturing and viewing video from v4l2 devices"
+HOMEPAGE="https://guvcview.sourceforge.net/"
+SRC_URI="https://downloads.sourceforge.net/project/${PN}/source/${PN}-src-${PV}.tar.bz2"
+S="${WORKDIR}/${PN}-src-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gsl pulseaudio qt6 +sdl sfml"
+REQUIRED_USE="
+	|| ( sdl sfml )
+"
+BDEPEND="
+	sys-devel/gettext
+	virtual/pkgconfig
+"
+RDEPEND="
+	media-libs/libpng:=
+	media-libs/libv4l
+	media-libs/portaudio
+	media-video/ffmpeg:=
+	virtual/libusb:1
+	virtual/udev
+	gsl? ( sci-libs/gsl:= )
+	pulseaudio? ( media-libs/libpulse )
+	qt6? ( dev-qt/qtbase:6[gui,widgets]	)
```

Comment:
So, there's always a GUI, and it's either Qt (6) or GTK 3.

Do upstream prefer either one of them? _Ideally_ in cases like this, where GUI is mandatory, we make one of them implied/default, and then we have a USE=gtk or whatever to change (so there would be no USE=qt6 if qt was preferred by upstream).

Reply:
Yes. Gtk3 as default (upstream).
So : only one useflag for Qt6 alternative.
Already done after a71r review.